src/shared/utils/roles/can-create-community.ts

```diff
@@ -0,0 +1,12 @@
+import { GetSiteResponse } from "lemmy-js-client";
+import { UserService } from "../../services";
+import { amAdmin } from "./am-admin";
+
+export function canCreateCommunity(
```

Comment:
If we're using a single function, a default subreddit may be better. Same for other similar files.

Reply:
I’m not sure I understand. 😅

Reply:
I meant default export lol. No idea where subreddit came from.